### DISCUSSION TITLE
Add dynamic refresh rate option

### DISCUSSION THREAD

**Discussion by demandred1687** [seerge/g-helper#2097] [Ideas]
AMD released in their newest chipset and graphic drivers the option to switch to dynamic refresh rate. (i think this is only confirmed for G14 2022 and 2023 version please see attached).  I find it a useful improvement if it could be incorporated in the g-helper screen as well. what i would most probably use it for - on battery switch to dynamic 60-120 and on charge 165hz. I guess it's a useful automation. :)
![2024-02-21 17 28 30](https://github.com/seerge/g-helper/assets/149145786/a7afe12f-89da-46b3-9c58-a84951d17ae2)

ps i tried searching for similar discussion and i apologize in advance if this was already discussed

**Comment by seerge** [comment]
@demandred1687 hello, I was already answering this question here https://github.com/seerge/g-helper/issues/2041

In short - it's not possible via windows api

**Reply by demandred1687** [reply]
aw. thanks for answering and love your work!
